docs/reference/chatbot-ollama.md

```diff
@@ -0,0 +1,141 @@
+- [:material-kubernetes: Deployment configuration](https://github.com/swiss-ai-center/chatbot-ollama-service/tree/main/kubernetes)
+- [:material-test-tube: Staging](https://chatbot-ollama-swiss-ai-center.kube.isc.heia-fr.ch/)
+- [:material-factory: Production (not available yet)](https://chatbot-ollama.swiss-ai-center.ch/)
+
+This service is a streamlit interface to interact with a large language model
+using the [ollama API](https://ollama.ai/) and langchain.
+
+Ollama is an open source project that aims to provide a simple and easy to use
+app to download and use LLM models like llama2 or mistral locally.
+
+On the [Ollama official website](https://ollama.ai/), you can find the list of
+available models and the link to the
+[github repository](https://github.com/jmorganca/ollama).
+
+Before using this service, you need to vectorize a pdf document using the
+[document-vectorizer service](/reference/services/document-vectorizer/).
+
+## Environment variables
+
+All environment variables are described in the
+[`.env`](https://github.com/swiss-ai-center/chatbot-ollama-service/blob/main/.env)
+file.
+
+The environment variables can be overwritten during the CI/CD pipeline described
+in the
+[`workflow.yml`](https://github.com/swiss-ai-center/chatbot-ollama-service/blob/main/.github/workflows/workflow.yml)
+GitHub workflow file.
+
+## Start the service locally with Python
+
+In the `services/chatbot-ollama` directory, start the service with the following
+commands.
+
+```sh
+# Generate the virtual environment
+python3 -m venv .venv
+
+# Activate the virtual environment
+source .venv/bin/activate
+
+# Install the requirements
+pip install /code[ui]
+```
+
+Start the application.
+
+```sh
+# Start the application
+streamlit run app/app.py --server.port 80
+```
+
+Access the service app on <http://localhost:80>.
+
+## Start the service locally with minikube and the Docker image hosted on GitHub
+
+Start the service with the following commands. This will start the service with
+the official Docker images that are hosted on GitHub.
+
+In the `chatbot-ollama` directory, start the service with the following
+commands.
+
+```sh
+# Start the chatbot-ollama
+kubectl apply \
+    -f kubernetes/chatbot-ollama.config-map.yml \
+    -f kubernetes/chatbot-ollama.stateful.yml \
+    -f kubernetes/chatbot-ollama.service.yml
+```
+
+Create a tunnel to access the Kubernetes cluster from the local machine. The
+terminal in which the tunnel is created must stay open.
+
+```sh
+# Open a tunnel to the Kubernetes cluster
+minikube tunnel --bind-address 127.0.0.1
+```
+
+Access the `chatbot-ollama` app on <http://localhost:9090/>.
+
+Then you must use the document-vectorizer service to vectorize a pdf document before using the chatbot-ollama app.
+See the [document-vectorizer service](/reference/services/document-vectorizer/#start-the-service-locally-with-minikube-and-the-docker-image-hosted-on-github) documentation to deploy it locally.
```

Comment:
You can use relative links (`../reference/services/document-vectorizer.md#start-the-service-locally-with-minikube-and-the-docker-image-hosted-on-github`) instead of absolute links. This allows to `CTRL + Click` in our IDEs/GitHub Web UI to directly access the link(s). Material for MkDocs works great with this.